docs/latency-limitations/latency-limitations.md

```diff
@@ -0,0 +1,37 @@
+# Latency-imposed limitations
+
+## etcd limitations
+
+A distance between etcd nodes in the multi-cloud environment of more than 600 km can be detrimental to cluster health. In a scenario like this, an average deployment time can double compared to a scenario with etcd nodes in different availability zones within the same cloud provider. Besides this, the total number of the etcd Slow Applies increases rapidly, and a Round-trip time varies from ~0.05s to ~0.2s, whereas in a single-cloud scenario with etcd nodes in a different AZs the range is from ~0.003s to ~0.025s. 
+
+In multi-cloud clusters, a request to a KubeAPI lasts from ~0.025s to ~0.25s. On the other hand, in a one-cloud scenario, they last from ~0.005s to ~0.025s.
+
+You can read more about this topic [here](https://www.berops.com/blog/evaluating-etcds-performance-in-multi-cloud).
```

Comment:
We can say here, that for distances above 600 km, the `etcd` deployment should be customized further by the user (and refer to the etcd docs for more info).